java-security/src/main/java/com/sap/cloud/security/token/x509/X509CertSelector.java

```diff
@@ -0,0 +1,11 @@
+package com.sap.cloud.security.token.x509;
+
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * Interface for accessing X509 certificates.
+ */
+public interface X509CertSelector {
+
+	String getCertificate(HttpServletRequest request);
```

Comment:
maybe we should offer a default create method, that checks whether its in CF environement and return CFx509CertSelector